os/net/mac/tsch/tsch-slot-operation.h

```diff
@@ -67,11 +67,10 @@ extern int tsch_current_burst_count;
  * The offset to the ASN and performs a hopping sequence lookup.
  *
  * \param asn A given ASN
- * \param channel_offset Link's channel offset
- * \param p Packet that can override the link's channel offset
+ * \param channel_offset Given channel offset
  * \return The resulting channel
  */
-uint8_t tsch_calculate_channel(struct tsch_asn_t *asn, uint16_t channel_offset, struct tsch_packet *p);
+uint8_t tsch_calculate_channel(struct tsch_asn_t *asn, uint16_t channel_offset);
```

Comment:
Looks like this can be made now static

Reply:
True - it makes sense to keep it in the .c file. I updated the PR.